augraphy/augmentations/dirtyrollers.py

```diff
@@ -12,26 +12,32 @@ class DirtyRollers(Augmentation):
     :param line_width_range: Pair of ints determining the range from which the
     width of a dirty roller line is sampled.
     :type line_width_range: tuple, optional
+    :param scanline_type: Types of scanline, use 0 for white background.
+    :type scanline_type: int, optional
     :param p: The probability this Augmentation will be applied.
     :type p: float, optional
     """
 
     def __init__(
         self,
         line_width_range=(8, 12),
+        scanline_type=0,
         p=1,
     ):
         """Constructor method"""
         super().__init__(p=p)
         self.line_width_range = line_width_range
+        self.scanline_type = scanline_type
 
     # Constructs a string representation of this Augmentation.
     def __repr__(self):
-        return f"DirtyRollers(line_width_range={self.line_width_range}, p={self.p})"
+        return f"DirtyRollers(line_width_range={self.line_width_range}, scanline_type={self.scanline_type}, p={self.p})"
 
     def apply_scanline_mask(self, img, mask, meta_mask):
-        if random.choice([True, False]):
+        # for dark background
+        if self.scanline_type:
```

Comment:
I added `scanline_type` argument so that user can choose the proper scanline method. This is due to scanline_mask_v2 doesn't work with white background.

With this equation in `apply_scanline_mask_v2` : `min(255, x + (x * (1 - (y / 100))))`, `x` would be the background and y will be the mask. So if background = white (value = 255), the output will be always 255 too.

For example, with white background:
left = scanline_v2, right = scanline_v1
![image](https://user-images.githubusercontent.com/52521979/146317765-bd265f0f-0205-4ab3-a2cd-a811a4bcc524.png)

So with white background, user should use self.scanline_type = 0.